run.sh

```
@@ -1,19 +1,395 @@
-echo "Open http://localhost:5173 in your browser"
-wait
\ No newline at end of file
+#  colours 
+RED='\033[0;31m'; GREEN='\033[0;32m'; YELLOW='\033[1;33m'
+BLUE='\033[0;34m'; BOLD='\033[1m'; NC='\033[0m'
+
+info()    { echo -e "${BLUE}[INFO]${NC}    $*"; }
+success() { echo -e "${GREEN}[OK]${NC}      $*"; }
+warn()    { echo -e "${YELLOW}[WARN]${NC}    $*"; }
+error()   { echo -e "${RED}[ERROR]${NC}   $*"; }
+header()  { echo -e "\n${BOLD}$*${NC}"; echo "$(printf '%.0s' {1..50})"; }
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)